src/tests/Common/Platform/platformdefines.cpp

```diff
@@ -375,24 +339,24 @@ HRESULT CbSysStringSize(ULONG cchSize, BOOL isByteLen, ULONG *result)
 
     // +2 for the null terminator
     // + DWORD_PTR to store the byte length of the string
-    int constant = sizeof(WCHAR) + sizeof(DWORD_PTR) + WIN32_ALLOC_ALIGN;
+    ULONG constant = sizeof(WCHAR) + sizeof(DWORD_PTR) + WIN32_ALLOC_ALIGN;
 
     if (isByteLen)
     {
-        if (SUCCEEDED(ULongAdd(constant, cchSize, result)))
+        if (ClrSafeInt<ULONG>::addition(constant, cchSize, *result))
         {
             *result = *result & ~WIN32_ALLOC_ALIGN;
-            return S_OK;
+            return NOERROR;
         }
     }
     else
     {
-        ULONG temp = 0; // should not use in-place addition in ULongAdd
-        if (SUCCEEDED(ULongMult(cchSize, sizeof(WCHAR), &temp)) &
-            SUCCEEDED(ULongAdd(temp, constant, result)))
+        ULONG temp = 0;
+        if (ClrSafeInt<ULONG>::multiply(cchSize, sizeof(WCHAR), temp) &&
+            ClrSafeInt<ULONG>::addition(temp, constant, *result))
```

Comment:
We do not have safemath header in tests here.
Keeping copies of code with **different modification** looks not good to me.

Reply:
I would leave the tests alone. The interop tests somewhat intentionally do not include the headers that coreclr itself is built against.

Reply:
The build for these test binaries are now generally using the same configuration as the coreclr build. This means we can occasionally use the product headers, but only if they are header only implementations. The issue here is we don't link against any product libraries so this can get tricky and make product reuse logic complicated.

For right now, I agree with @jkotas, but do agree reusing more of these utility functions would be beneficial.

Reply:
Now there's no problem of duplication since one definition is gone.